src/templates/qa_hpwh.mdx

```diff
@@ -51,221 +50,514 @@
+      efficiency specifications of the HPWH but also the fuel type of the existing water
+      heater the HPWH is replacing. For jurisdictions where this applies, please provide
+      a photo of the nameplate showing the model number,
+      fuel type, and the serial number of the old water heater.
+    </Photo>
+
+    <br></br>
+
+    ---
+
+
+    <Collapsible header="HPWH Selection – Instructions">
+      The efficiency of HPWHs is profoundly affected by the way hot water is used and thus,
+      selecting the right sized HPWH is important to user satisfaction.
+      Most HPWHs on the market are the type with a small heat pump (e.g., 550-watt compressor)
+      on top of a storage water heater tank. These products have a refrigerant loop
+      heat exchanger wrapped around and integrated into the tank as well as electric
+      resistance elements to provide supplemental/back up heat when hot water demands
+      deplete the tank faster than the small heat pump can replenish the tank with heat.
+      An illustration of how a HPWH works is shown below.
+
+      <Figure src="/images/hpwh/hpwh_components.jpeg">
+        The components in a common HPWH (BASC 2015)
+      </Figure>
+
+      The heat pump can deliver the same heat into the water in the tank using 1/3 to ¼ of
+      the electricity of the electric resistance elements but requires more time to do this.
+      The HPWH will operate less efficiently when the electric resistance elements are used
+      to provide supplemental heat. If the typical hot water demands are mostly satisfied by
+      the stored hot water in the tank, then it does not affect user comfort whether the HPWH
+      replenishes the heat in the tank quickly but less efficiently or slower but much more
+      efficiently. Water heater manufacturers list how much hot water a water heater can
+      supply during a period of high demand using a metric called the first-hour rating (FHR).
+      The first-hour rating (FHR) is a hot water delivery calculation that factors in both
+      the water heater volume and the heating rate. Since the heat pump heats the water
+      slowly, the tank volume of hot water will largely determine the FHR in a HPWH. This is
+      why selecting a HPWH with a tank large enough to satisfy the peak hour of hot water
+      demand is important in selecting the correct HPWH for the needs of the house.
+
+      To estimate your peak hour demand:
+      - Determine what time of day (morning, noon, evening) you use the highest amount of
+      hot water in your home. Keep in mind the number of people living in your home.
+      - Use the worksheet below to estimate your maximum usage of hot water during this
+      one-hour period of the day—this is your peak hour demand. Note: the worksheet does not
+      estimate total daily hot water usage.
+
+      * Image*: Worksheet for Estimating Peak Hour Demand/First Hour Rating
```

Comment:
Oh oh, do we have these images?  I thought we had them before.

Reply:
I haven't noticed this before. I've made no changes to the HPWH Selection – Instructions collapsable